asm/f64/axpyinc_amd64.s

```diff
@@ -131,8 +131,8 @@ tail_two: // do {
 	DECQ LEN
 	JNZ  tail_two                      // } while --LEN > 0
 
-	CMPQ TAIL, $0
-	JE   end      // if TAIL == 0 { goto end }
+	ANDQ $1, TAIL
```

Comment:
Can you explain the bug and fix in the commit message please. This seems odd; is TAIL never even?